golem_sci/implementation.py

```diff
@@ -599,7 +599,14 @@ def _pull_eth_subscription_events(self) -> None:
                     self._on_event(t, sub.cb)
                 sub.last_pulled_block = self._confirmed_block
```

Comment:
Please note that this won't be updated then, thus this error will keep happening.

Reply:
amended here: https://github.com/golemfactory/golem-smart-contracts-interface/pull/71/files#diff-8bd37e16f9c49ec8176ea37b4bdb96cdR610